src/manifest.c

```diff
@@ -633,7 +633,6 @@ struct manifest *load_mom(int version)
 		printf("Failed to load %d MoM manifest\n", version);
 		goto out;
 	}
-	retried = false;
```

Comment:
The retry code in load_mom and such functions is different then the top level code since it assumes it is dealing with IO errors or system errors, which will not benefit from timeouts. If retrieving a manifest fails - i.e due to curl/network errors - then timeouts will be hit and allow for more time for the network to come up if it went down.
This code can be refactored to support both cases more fluidly and reduce these kinds of errors, but that will be a significant rework. Therefore, I opened an issue to track the general problem of retries https://github.com/clearlinux/swupd-client/issues/211 and we can plan that out separately.